README.md

```diff
@@ -6,22 +6,36 @@ This repository contains resources for deploying the Deep Log Inspection system
 
 This project is part of [FIWARE][1]. Credits to [monasca-docker][2] and [docker-elk][3].
 
-Quick Start
------------
+## Quick Start
```

Comment:
Keep here just a reference to the quickstart.md to remove the duplicated content

Reply:
done